pkgs/applications/video/kodi/unwrapped.nix

```diff
@@ -87,22 +87,18 @@ let
 
 in stdenv.mkDerivation (finalAttrs: {
     pname = "kodi";
-    version = "21.1";
+    version = "21.2";
     kodiReleaseName = "Omega";
 
     src = fetchFromGitHub {
       owner = "xbmc";
       repo  = "xbmc";
       rev   = "${finalAttrs.version}-${finalAttrs.kodiReleaseName}";
-      hash  = "sha256-NjId1T1cw9dl0Fx1QDsijiN1VUpuQ/EFl1kxWSESCR4=";
+      hash  = "sha256-RdTJcq6FPerQx05dU3r8iyaorT4L7162hg5RdywsA88=";
     };
 
     patches = [
       ./no-python-lib.patch
```

Comment:
Do we still need this patch? I see in `git blame`, the only reason it was introduced is Python 3.12 support.
Kodi builds fine without it now.

Reply:
great, thanks! this is why comments on patches are good 😊 

i will remove, rebuild, and push